uaclient/tests/test_status.py

```diff
@@ -574,9 +575,19 @@ def test_root_followed_by_nonroot(
 
         # Run as regular user and confirm that we see the result from
         # last time we called .status()
+        # Since machine is attached then diff is found
         m_getuid.return_value = 1000
+        cfg = FakeConfig(root_mode=False)
         after = status.status(cfg=cfg)
```

Comment:
I think we also need a check to see if `root` and `non-root` status are the same

Reply:
Makes sense.